src/simularium/RemoteSimulator.ts

```diff
@@ -12,15 +12,8 @@ import { ISimulator } from "./ISimulator";
 import { TrajectoryFileInfoV2, VisDataMessage } from "./types";
 import { TrajectoryType } from "../constants";
 
-const enum PlayBackType {
-    ID_LIVE_SIMULATION = 0,
-    ID_PRE_RUN_SIMULATION = 1,
-    ID_TRAJECTORY_FILE_PLAYBACK = 2,
-    // insert new values here before LENGTH
-    LENGTH,
-}
```

Comment:
Why is this enum no longer needed? Is it only used by Simularium engine?

Reply:
Correct! Octopus doesn't use this